setup.py

```diff
@@ -119,6 +119,7 @@
             'object_store_objects = cfme.storage.object_store_object:ObjectStoreObjectCollection',
             ('object_store_containers = '
                 'cfme.storage.object_store_container:ObjectStoreContainerCollection'),
+            'images = cfme.containers.image:ImageCollection',
```

Comment:
**Required**

Use something more specific, like `container_images`, since there are other types of image collections.

Reply:
Done